pages/main.scss

```diff
@@ -115,11 +117,61 @@ p {
       .right .carousel-icons { right: 0; }
     }
   }
+
   .services {
-    #brief {
-      width: 109px;
-      height: auto;
-      margin: auto;
+    $unit-size: 64px;
+    .rolodex {
+      height: $unit-size;
+      position: relative;
+      .rolodex-control {
+        height: 100%;
+        position: absolute;
+        padding-top: 16px;
+        top: 0;
+      }
+      .left {left: 0;}
+      .right {right: 0;}
+      .rolodex-inner {
+        height: 100%;
+        width: $unit-size * 3.6;
+        margin: auto;
+        overflow-x: hidden;
+        .rolodex-list {
+          white-space: nowrap;
+          position: relative;
+          top: 0;
+          left: 0;
+        }
+        .rolodex-item {
+          position: absolute;
+          top: 0;
+          opacity: 0.2;
+        }
+        .rolodex-item.active {
+          opacity: 1;
+          transition: opacity 1.5s;
+        }
+        .svg-icon {
+          width: $unit-size;
+          height: $unit-size;
+        }
+      }
+    }
+    .rolodex-content {
+      height: $unit-size * 2;
+      .rolodex-content-item {
+        height: 0;
+        opacity: 0;
+        transition: opacity 0.5s;
+      }
+      .rolodex-content-item.active {
+        opacity: 1;
+        transition: opacity 0.5s;
+      }
+      p {
+        position: relative;
+        top: 0; left: 0;
+      }
```

Comment:
The css for this seems to be quite complex, can you talk me through it when you have a mo?